src/dsverifier.cpp

```diff
@@ -156,6 +157,10 @@ double desired_quantization_limit = 0.0;
 bool show_counterexample_data = false;
 bool preprocess = false;
 
+/* MACROS */
```

Comment:
Can you add some description about these macros?

Reply:
DONE